x-pack/plugin/text-structure/src/main/java/org/elasticsearch/xpack/textstructure/structurefinder/TimestampFormatFinder.java

```diff
@@ -288,6 +289,182 @@ public final class TimestampFormatFinder {
         )
     );
 
+    /**
+     * The first match in this list will be chosen, so it needs to be ordered
+     * such that more generic patterns come after more specific patterns.
+     */
+    static final List<CandidateTimestampFormat> ORDERED_CANDIDATE_FORMATS_ECS_V1 = Arrays.asList(
```

Comment:
From what I can tell, this list is EXACTLY the same as the non-ecs one except some new timestamps for catalina are prepended. 

Do we really need a whole new list here?